docker-compose.yml

```diff
@@ -1,6 +1,42 @@
 version: "3.0"
 
 services:
+  core-catalog-writer:
+    container_name: core-catalog-writer
+    image: openjdk:17-ea-16-jdk
+    volumes:
+      - ./services/core-catalog-writer/build/libs/core-catalog-writer.jar:/core-catalog-writer.jar
+    command: [ "java", "-jar", "core-catalog-writer.jar" ]
+    ports:
+      - "8083:8083"
+    depends_on:
+      - core-catalog-postgresql
```

Comment:
Also just for a note I want to share that you can define a health check for a docker compose service and then wait until it is healthy, not just started. It can be useful.